### PR TITLE
[4.0] Fix featured view and add grouped buttons

### PR DESCRIPTION
**Pull request by bembelimen**
### Summary of Changes
This PR continues the grouping of action buttons in the backend (see https://github.com/joomla/joomla-cms/pull/22832, https://github.com/joomla/joomla-cms/pull/22393, https://github.com/joomla/joomla-cms/pull/23337 and https://github.com/joomla/joomla-cms/pull/23336). This time the featured buttons.

### Testing Instructions
Go to the featured articles in the backend and check the toolbar. There should be a "Change status" dropdown now which activates when you select a featured item.

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on a793d3f388a123ae160ab73a9319986f9bb0bdc5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23459">issues.joomla.org/tracker/joomla-cms/23459</a>.</sub>

File: administrator/components/com_content/tmpl/featured/default_stage_footer.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_content
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Factory;
```
Comment (Quy) [inline administrator/components/com_content/tmpl/featured/default_stage_footer.php:11]: Remove?

**Comment by PhocaCz** [comment]
I have tested this item :white_check_mark: successfully on a793d3f388a123ae160ab73a9319986f9bb0bdc5<br /><br />Tested on Joomla! 4.0.0-alpha7-dev

OK<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23459">issues.joomla.org/tracker/joomla-cms/23459</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**